kroxylicious-kms-provider-aws-kms/src/test/scripts/echoing_http_server.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
```

Comment:
Did not expect to see a couple of python scripts here. 

Should we document that in the dev guide? Maybe not as python3 is practically ubiquitous these days....

Reply:
I wasn't too happy with leaning on curl to generate the know good.  But it seemed like the least worst option available.  Since developers won't generally need to regenerate the known good, I think it is okay to leave it out of the dev guide.